CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ add_custom_target(rclcs ALL
 else()
 add_custom_target(rclcs ALL
 	COMMAND 
-	"msbuild" rclcs.csproj /property:OutputPath=${CMAKE_BINARY_DIR} /property:AssemblyName=rclcs /property:AllowUnsafeBlocks=true /property:Platform=x64 /property:GenerateDocumentation=true /property:DocumentationFile="rclcs.xml"
+	"msbuild" rclcs.csproj /property:OutputPath=${CMAKE_BINARY_DIR} /property:AssemblyName=rclcs /property:Configuration=Release /property:Platform=x64 /property:AllowUnsafeBlocks=true /property:DocumentationFile="rclcs.xml"
```

Comment:
Is it really necessary to built this as a release build ? Doesn't this make debugging harder?